src/Hooks/HookManager.php

```diff
@@ -87,18 +93,18 @@ public function on(string $name, callable $function)
     /**
      * Fires the hook
      * @param string $name
-     * @param array $args
-     * @throws \Quantum\Exceptions\HookException
+     * @param array|null $args
+     * @throws HookException
      */
     public function fire(string $name, array $args = null)
```

Comment:
Add '?' before the $args type because null can be passed according to the dock block.